packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/translations/Messages_fr_FR.json

```
@@ -0,0 +1,10 @@
+{
+  "telephony_white_label_manager_title": "Manager marque blanche",
+  "telephony_white_label_manager_descrition": "Le manager marque blanche vous offre une interface personnalisable aux couleurs de votre société pour gérer les services Telecom OVHcloud.",
+  "telephony_white_label_manager_checkbox_label": "Je souhaite réinitialiser le mot de passe de mon manager marque blanche.",
```

i guess it's better with a upper capital lettrer, non ?
```suggestion
  "telephony_white_label_manager_checkbox_label": "Je souhaite réinitialiser le mot de passe de mon Manager marque blanche.",
```

---

I fixed.

---

I fixed.